### PR TITLE
ARUHA-2372 Store event disk size stats from nakadi

### PR DESCRIPTION
**Pull request by antban**
ARUHA-2372 Store event disk size stats from nakadi

### CODE REVIEW

File: src/main/resources/disk_usage_event_type.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      ],
+      "writers": [
+        {
+          "data_type": "auth_data_type_placeholder",
+          "value": "auth_value_placeholder"
+        }
+      ]
+    }
+  }
+]
```
Comment (v-stepanov) [inline src/main/resources/disk_usage_event_type.json]: endline

**Comment by v-stepanov** [comment]
@antban code looks good.
But I think `KafkaTopicRepository.getSizeStats()` and `DiskUsageStatsJob` need tests.

File: src/main/java/org/zalando/nakadi/service/job/DiskUsageStatsJob.java [hunk 2]
```diff
@@ -71,6 +72,13 @@ public void dumpDiskStats() {
     }
 
     private void dumpDiskStatsLocked() {
+        final Map<String, Long> eventTypeSize = loadDiskUsage();
+        // Stats are here.
```
Comment (v-stepanov) [inline src/main/java/org/zalando/nakadi/service/job/DiskUsageStatsJob.java]: What does this comment mean?

File: src/test/java/org/zalando/nakadi/service/job/DiskUsageStatsJobTest.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        final Map<String, Long> expectedResult = new HashMap<>();
+        expectedResult.put("et1", 5L);
+        expectedResult.put("et2", 18L);
+        expectedResult.put("et3", 513L);
+        expectedResult.put("et4", 1836L);
+
+        final Map<String, Long> actualResult = job.loadDiskUsage();
+        assertEquals(expectedResult, actualResult);
+    }
+}
```
Comment (v-stepanov) [inline src/test/java/org/zalando/nakadi/service/job/DiskUsageStatsJobTest.java]: ping

File: src/main/java/org/zalando/nakadi/repository/kafka/KafkaZookeeper.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
+package org.zalando.nakadi.repository.kafka;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.zalando.nakadi.repository.zookeeper.ZooKeeperHolder;
+
+import java.util.List;
+
+public class KafkaZookeeper {
```
Comment (v-stepanov) [inline src/main/java/org/zalando/nakadi/repository/kafka/KafkaZookeeper.java]: Looks more like BubukuZookeeper :)

File: src/test/java/org/zalando/nakadi/service/job/DiskUsageStatsJobTest.java (hunk 3 above)
Comment (v-stepanov) [reply src/test/java/org/zalando/nakadi/service/job/DiskUsageStatsJobTest.java]: you can configure your IDE to add that automatically

**Comment by v-stepanov** [comment]
Everything looks good. Fix that super major problem with endline at the end of file and I will approve.

**Comment by v-stepanov** [comment]
👍

**Comment by antban** [comment]
:+

**Comment by antban** [comment]
:+1:

**Pull request merged**